src/firefighter/slack/signals/create_incident_conversation.py

```diff
@@ -37,13 +38,15 @@
 @receiver(signal=create_incident_conversation)
 def create_incident_slack_conversation(
     incident: Incident,
+    source_channel: IncidentChannel,
```

Comment:
In `trigger_incident_workflow`, it's `Any`, so we need to check that it's an `IncidentChannel`, and probably accept None by default.